transcriptTransformationScript.js

```diff
@@ -35,6 +35,82 @@ function addSeparationAttribute(currEntry) {
     }
 }
 
+function convertColumnsWidthToString(items) {
+    for (var column of items["columns"]) {
+        if (column.hasOwnProperty("width")) {
+            column["width"] = column["width"] + "";
+        }
+    }
+}
+// Handler 3
+function convertNumbersToString(currEntry) {
+    if (currEntry['type'] === 'message') {
+        if (currEntry.hasOwnProperty("attachments")) {
+            for (var attachment of currEntry["attachments"]) {
+                if (attachment.hasOwnProperty("content") && attachment["content"].hasOwnProperty("body")) {
+                    for (var bodyItem of attachment["content"]["body"]) {
+                        if (bodyItem.hasOwnProperty("columns")) {
+                            convertColumnsWidthToString(bodyItem);
+                        } else if (bodyItem.hasOwnProperty("items")) {
+                            for (var item of bodyItem["items"]) {
+                                if (item.hasOwnProperty("columns")) {
+                                    convertColumnsWidthToString(item);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+// Handler 4
+function convertColumnAttributesToCamelCase(currEntry) {
+    if (currEntry['type'] === 'message') {
+        if (currEntry.hasOwnProperty("attachments")) {
+            const attachments = currEntry["attachments"];
+            attachments.forEach(attachment => {
+                if (attachment.hasOwnProperty("content")) {
+                    const content = attachment["content"];
+                    if (content.hasOwnProperty("body")) {
+                        const contentBody = content["body"][0];
+                        const bodyItems = contentBody["items"];
```

Comment:
Does `content["body"][0]` always have `items` attribute?

Reply:
For the cases i had, it does, but you are right; an if statement should be added 👍
I'll add it